kod/object/active/holder/nomoveon/battler/player.kod

```diff
@@ -321,6 +321,7 @@ resources:
    player_desc_enchanted_none = "%q"
    player_desc_enchanted = "%s has %s%s.\n\n%s"
    
+   player_not_hungry_sw = "Still feeling a surge of vitality, you are uninterested in food or drink right now."
```

Comment:
This will move to SecondWind (see below).  This will have the advantage of not requiring an awkward "_sw" suffix, which is a sign that this doesn't really belong here.